.github/workflows/xcode-versions-build.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       max-parallel: 3
       matrix:
         variant: [cocoapods, spm]
-        xcode: [14.3.1, 14.2, 14.1]
```

Comment:
[q]: does this mean merchants won't be able ti integrate using these old-ish Xcode versions?

Reply:
No, just that we are tesing only on 14.3.1, it should be enough to say it "works on Xcode 14" since these are all point releases of the same Xcode version.
Thoughts @jnewc @BorisNikolic (this was an old suggestion about wasted build times)

Reply:
I don't see a reason not to do it. Xcode 15 was released on September 18th. 4 months ago... And the diff between 14.3.1 and 14.2 and 14.1 is very minor.

Reply:
Agree that 14.3.1 is enough 👍